media-proxy/tests/memif_test_tx.c

```diff
@@ -44,7 +44,7 @@ int build_frames(app_context_t* p_app_ctx, memif_buffer_t* tx_bufs, uint32_t buf
                 rewind(p_app_ctx->video_fd);
                 ret = 0;
             } else {
-                perror("Error when read frame file");
+                perror("End of file. Read frame resulted in ");
```

Comment:
Resulted in what?

Reply:
The `perror` prints the state of last error. If it is success it will print `End of file. Read frame resulted in: success`.